docs/projects/daq2/index.rst

```diff
@@ -0,0 +1,554 @@
+   :widths: 35 35 30
+   :header-rows: 1
+
+   * - Evaluation board
+     - Carrier
+     - FMC slot
+   * - :adi:`AD-FMCDAQ2-EBZ <EVAL-AD-FMCDAQ2-EBZ>`
+     - :xilinx:`KC705`
+     - FMC HPC
+   * -
+     - :xilinx:`KCU105`
+     - FMC HPC
+   * -
+     - :xilinx:`ZC706`
+     - FMC HPC
+   * -
+     - :xilinx:`ZCU102`
+     - FMC HPC0
+
+Block design
+-------------------------------------------------------------------------------
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The data path and clock domains are depicted in the below diagram:
+
+.. image:: daq2_xilinx_block_diagram.svg
+   :width: 1000
+   :align: center
+   :alt: DAQ2 block diagram
+
+.. collapsible:: Click here for details on the block diagram modules
+
+   .. list-table::
+      :widths: 10 20 35 35
+      :header-rows: 1
+
+      * - Block name
+        - IP name
+        - Documentation
+        - Additional info
+      * - AXI_ADXCVR
+        - :git-hdl:`axi_adxcvr <library/xilinx/axi_adxcvr>`
+        - :ref:`axi_adxcvr`
+        - 2 instances, one for Rx and one for Tx
+      * - AXI_DMAC
+        - :git-hdl:`axi_dmac <library/axi_dmac>`
+        - :ref:`axi_dmac`
+        - 2 instances, one for Rx and one for Tx
+      * - RX JESD LINK
+        - axi_ad9680_jesd
+        - :ref:`axi_jesd204_rx`
+        - Instantiated by ``adi_axi_jesd204_rx_create`` procedure
+      * - RX JESD TPL
+        - axi_ad9680_core
+        - :ref:`ad_ip_jesd204_tpl_adc`
+        - Instantiated by ``adi_tpl_jesd204_rx_create`` procedure
+      * - TX JESD LINK
+        - axi_ad9144_jesd
+        - :ref:`axi_jesd204_tx`
+        - Instantiaded by ``adi_axi_jesd204_tx_create`` procedure
+      * - TX JESD TPL
+        - axi_ad9144_core
+        - :ref:`ad_ip_jesd204_tpl_dac`
+        - Instantiated by ``adi_tpl_jesd204_tx_create`` procedure
+      * - UTIL_UPACK
+        - :git-hdl:`util_upack2 <library/util_pack/util_upack2>`
+        - :ref:`util_upack2`
+        - ---
+      * - UTIL_CPACK
+        - :git-hdl:`util_upack2 <library/util_pack/util_cpack2>`
+        - :ref:`util_cpack2`
+        - ---
+      * - UTIL_ADXCVR
+        - :git-hdl:`util_adxcvr <library/xilinx/util_adxcvr>`
+        - :ref:`util_adxcvr`
+        - ---
+      * - DATA_OFFLOAD
+        - :git-hdl:`data_offload <library/data_offload>`
+        - :ref:`data_offload`
+        - 2 instances, one for Rx and one for Tx
+
+The reference design is a processor based (ARM or Microblaze) embedded system. A
+functional block diagram of the system is given above. The shared transceivers
+are followed by the individual JESD204B and ADC/DAC IP cores. The cores are
+programmable through an AXI-lite interface.
+
+The digital interface consists of 4 transmit and 4 receive lanes running at
+10Gbps, by default. The transceivers interface the ADC/DAC cores at
+128bits @250MHz. The data is sent or received based on the configuration of
+separate transmit and receive chains.
+
```

Comment:
```suggestion

Evaluation board block diagram
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The data path and clock domains in the evaluation board are depicted in the below diagram:

.. image:: daq2_board_block_diagram.svg
   :width: 1000
   :align: center
   :alt: DAQ2 evaluation board block diagram
```

Something of the sort, not necessarily 1:1 like this.
And to add this block diagram https://wiki.analog.com/resources/fpga/peripherals/jesd204/tutorial/hdl_xilinx#ad-fmcdaq2-ebz_block_diagram